.github/workflows/proto.yml

```diff
@@ -0,0 +1,30 @@
+name: buf
+on:
+  pull_request:
+  push:
+    branches:
+        - main
+jobs:
+  build-lint-push-protos:
+    runs-on: ubuntu-latest
+    environment: BUF
```

Comment:
Is this used anywhere? I believe it can be deleted.

Reply:
Yes, the BUF_TOKEN secret is attached to this environment. It gives the "deployment" below